README.md

```diff
@@ -92,6 +92,7 @@ You can run every example with zero setup as an "ANONYMOUS" Neptune user (no reg
 | PyTorch Lightning | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/model-training/pytorch-lightning) | [![neptune]](https://app.neptune.ai/o/common/org/pytorch-lightning-integration/e/PTL-11/dashboard/simple-6ff16e4c-c529-4c63-b437-dfb883131793) | [![github]](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/pytorch-lightning/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/integrations-and-supported-tools/pytorch-lightning/notebooks/Neptune_PyTorch_Lightning.ipynb) |
 | scikit-learn | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/model-training/sklearn) | [![neptune]](https://app.neptune.ai/o/common/org/sklearn-integration/e/SKLEAR-97/all?path=rfr_summary%2Fdiagnostics_charts&attribute=feature_importance) | [![github]](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/sklearn/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb) |
 | skorch | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/model-training/skorch) | | | |
+| 🤗 Transformers | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/model-training/hugging-face) | [![neptune]](https://app.neptune.ai/o/showcase/org/project-text-summarization-hf/e/PROJ-138/dashboard/Custom-Dashboard-97370bc5-ee32-48ce-a630-fc806a370e13)  |  | [![colab]](https://colab.research.google.com/drive/1bJReZxX7fN_ZM74uSPKFuix5WlDRrKrv?usp=sharing) |
```

Comment:
@kshitij12345 - this will need to be updated to point to notebook in github

Reply:
Updated.

Reply:
@kshitij12345 the notebook path should look like this
https://colab.research.google.com/github/neptune-ai/examples/blob/master/integrations-and-supported-tools/transformers/notebooks/Neptune_Transformers.ipynb
This ensures that colab always points to the latest notebook in git

Reply:
Sorry. Forgot to push the change.